localstack/services/awslambda/invocation/lambda_service.py

```diff
@@ -247,6 +247,12 @@ def invoke(
         except ValueError:
             version = function.versions.get(version_qualifier)
             state = version and version.config.state.state
+            # TODO: make such developer hints optional or remove after initial v2 transition period
+            LOG.warning(
+                "Lambda functions are created and updated asynchronously in the new lambda provider. "
+                "Please wait until the function is in the Active state before invoking. Check out "
+                "https://docs.localstack.cloud/references/lambda-provider-v2"
```

Comment:
Love this


I'd go a step further and show them a sample like `aws lambda wait function-active-v2 --function-name <actual-fn-name>`. This obviously won't apply to all use cases (e.g. SDKs) but might give them a good starting point, even if the docs are broken for some reason.

Reply:
Good idea to demonstrate the waiter solution as concretely as possible. Added an example:
<img width="1129" alt="Screenshot 2023-03-28 at 21 56 04" src="https://user-images.githubusercontent.com/8161292/228352537-89038f52-30d7-4e4a-8fb3-a441e723adaf.png">